pkg/northbound/gnmi/subscribe.go

```diff
@@ -326,7 +326,8 @@ func buildSubscribeResponse(notification *gnmi.Notification, target string) *gnm
 	response := &gnmi.SubscribeResponse{
 		Response: responseUpdate,
 	}
-	if _, ok := manager.GetManager().DeviceStore.Cache[device.ID(target)]; !ok {
+	_, err := manager.GetManager().DeviceStore.Get(device.ID(target))
+	if err != nil && status.Convert(err).Code() == codes.NotFound {
```

Comment:
maybe handle the `err` for other codes too

Reply:
done